src/Directory.Build.props

```diff
@@ -23,6 +23,14 @@
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <PackageReadmeFile Condition="'$(IsPackable)' == 'true'">README.md</PackageReadmeFile>
+
+    <!--
+      Strong name signing of netstandard assemblies for better cross-runtime compatibility
```

Comment:
Assembly for net6 TFM is also signed now.

Reply:
True, it is however only signing the netstandard assemblies that is done for better cross-runtime compatibility.